packages/app-project/stores/Store.spec.js

```diff
@@ -56,18 +56,14 @@ describe('Stores > Store', function () {
   })
 
   describe('when a user signs out', function () {
-    it('should reset the user project preferences and stats', function () {
+    it('should reset the user project preferences and sessionCount', function () {
```

Comment:
`sessionCount` was previously tied up with user stats in app-project's store. I've completely separate the two in this PR, so there are lots of spec files changes to accommodate.

Session count is used to display the AuthenticationInvitation component when a user is not signed in. It's also used to refetch a user's assigned workflow id every 5 classifications they submit. No effect on user classification stats.